### PR TITLE
feat: electron building logic

### PR DESCRIPTION
**Pull request by alvinhui**
- Add logs to local file；
- Refactor build logic.

### CODE REVIEW

**Comment by alvinhui** [comment]
> This work adds `log` functionality.

Just updated change notes.

File: tools/iceworks-app/gulpfile.js [hunk 1]
```diff
@@ -2,58 +2,111 @@ const os = require('os');
...
+    await filecopy(
+      'app/*',
+      'build'
+    );
+  }
+
+  async function setPackage() {
+    const packageJSONFileName = 'package.json';
+    const packageJSONPath = path.join(__dirname, packageJSONFileName);
+    const packageJSON = JSON.parse((await readFileAsync(packageJSONPath)).toString());
```
Comment (zhaofeng-shu33) [inline tools/iceworks-app/gulpfile.js]: duplicate with Line 8 `const packageJSON = require('./package.json');` ?

Comment (alvinhui) [reply tools/iceworks-app/gulpfile.js]: I have deleted `const packageJSON = require('./package.json');`

Comment (zhaofeng-shu33) [reply tools/iceworks-app/gulpfile.js]: OK

**Pull request merged**